File: rest-book-store/csharp/starter/RestBookStore.cs
using System;
using System.Collections.Generic;
using Xunit;

namespace RestBookStore
{
    public class Book
    {
        Book()
        { }
    }

    /// <summary>
    /// Path: /api/books
    /// </summary>
    public class BooksController
    {

        public void AddBook()
        {
            throw new NotImplementedException();
        }

        public void GetBook()
        {
            throw new NotImplementedException();
        }

        public void GetAllBooks()
        {
            throw new NotImplementedException();
        }
    }

    public class SampleSingleFile
    {
        [Fact]
        public void Test1()
        {
            var controller = new BooksController();

            Assert.Throws<NotImplementedException>(() => controller.GetAllBooks());
        }
    }
}

File: rest-book-store/java/starter/RestBookStore.java
import org.junit.Rule;
import org.junit.Test;
import org.junit.rules.ExpectedException;

class BooksController {

    public void addBook() {
        throw new UnsupportedOperationException();
    }

    public void getBook() {
        throw new UnsupportedOperationException();
    }

    public void getAllBooks() {
        throw new UnsupportedOperationException();
    }

}

public class RestBookStore {

    @Rule
    public ExpectedException exception = ExpectedException.none();

    @Test
    public void Test1() {
        var controller = new BooksController();

        exception.expect(UnsupportedOperationException.class);
        controller.addBook();
    }

}